include/uapi/sound/sof/abi.h

```diff
@@ -27,7 +27,7 @@
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
 #define SOF_ABI_MINOR 18
-#define SOF_ABI_PATCH 0
+#define SOF_ABI_PATCH 2
```

Comment:
NAK on this. This is an addition to the ABI so needs to bump MINOR version.

Reply:
> NAK on this. This is an addition to the ABI so needs to bump MINOR version.

@kv2019i Thanks for the reminder, I will change it to MINOR version.